SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKLogParser.cpp

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialGDKLogParser.h"
+
+#include "Algo/AllOf.h"
+#include "Dialogs/CustomDialog.h"
+#include "Editor.h"
+#include "Internationalization/Regex.h"
+#include "Widgets/Input/SHyperlink.h"
+
+#define LOCTEXT_NAMESPACE "SpatialGDKLogParser"
+
+static void ShowMissingSchemaDialog(const TSet<FString>& MissingSchemaPaths)
+{
+	struct Local
```

Comment:
I'm a bit worried about compiler support of this kind of construction (struct defined in a function).
Maybe every compiler caught up, but there was a time where only MSVC supported it.

Reply:
Haven't really had an issue with this during the last 3 years but this struct doesn't have to be inside of a function, it's more of a nice touch. Do you think we should move it outside?

Reply:
I read a bit more, apparently it's not actively discouraged, maybe the error was only related to me messing with templates at the time.